src/main/resources/migrations/00044_add_payouts_foreign_key_to_transaction.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_payouts_foreign_key_to_transaction
+ALTER TABLE transaction ADD COLUMN gateway_payout_id VARCHAR(50) REFERENCES payouts(gateway_payout_id);
+--rollback ALTER TABLE transaction DROP COLUMN gateway_payout_id; 
+
+--changeset uk.gov.pay:create_index_on_transaction_gateway_payout_id
+CREATE INDEX transaction_gateway_payout_id_idx ON transaction(gateway_payout_id);
```

Comment:
Would recommend `create index concurrently` so table won't be locked when building index. Although there is no data in the column it will still lock table for few millis.